gitbook/tables/clefs.html

```diff
@@ -2463,189 +2463,189 @@ <h2 id="recommended-ligatures">Recommended ligatures</h2>
 </thead>
 <tbody>
 <tr>
-<td style="text-align:center"><span class="bravura_large">&#xF4E3;</span></td>
```

Comment:
I see that it probably would have been better to make these comments in the corresponding .md files. I will leave them here though as this is where I found the issues by testing the .html files locally, rather than duplicating comments.